### PR TITLE
Update adc.c for stm32h753 microcontroller

### PR DESCRIPTION
**Pull request by RasailiTilak**

### CODE REVIEW

**Comment by RasailiTilak** [comment]
I am currently working on the stm32h753 microcontroller with the mcropython. i want to commit that to the micropython it will be help for my further project as well as for other too. thank you.

File: ports/stm32/adc.c [hunk 1]
```diff
@@ -171,7 +171,7 @@
     defined(STM32H743xx) || defined(STM32H747xx) || \
     defined(STM32H7A3xx) || defined(STM32H7A3xxQ) || \
     defined(STM32H7B3xx) || defined(STM32H7B3xxQ) || \
-    defined(STM32H750xx)
+    defined(STM32H750xx) || defined(STM32H753xx)
```
Comment (dpgeorge) [inline ports/stm32/adc.c:174]: Is this the only change you needed to make to get the STM32H753 working?

**Comment by RasailiTilak** [comment]
I am updating, i am preparing for other update.

**Comment by dpgeorge** [comment]
I'll close this in favour of #15516.

**Pull request closed without merging**